Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs

```diff
@@ -476,6 +476,90 @@ public IEnumerator ManipulationHandlerOneHandMoveFar()
 
         }
 
+
+        /// <summary>
+        /// This tests the one hand near rotation and applying different rotation constraints to the object.
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator ManipulationHandlerOneHandRotateWithConstraint()
+        {
+            // set up cube with manipulation handler
+            var testObject = GameObject.CreatePrimitive(PrimitiveType.Cube);
+            testObject.transform.localScale = Vector3.one * 0.2f;
+            Vector3 initialObjectPosition = new Vector3(0f, 0f, 1f);
+            testObject.transform.position = initialObjectPosition;
+            var manipHandler = testObject.AddComponent<ManipulationHandler>();
+            manipHandler.HostTransform = testObject.transform;
+            manipHandler.SmoothingActive = false;
+            manipHandler.ManipulationType = ManipulationHandler.HandMovementType.OneHandedOnly;
+            manipHandler.OneHandRotationModeFar = ManipulationHandler.RotateInOneHandType.RotateAboutGrabPoint;
+            manipHandler.ReleaseBehavior = 0;
+            manipHandler.AllowFarManipulation = false;
+
+            // add near interaction grabbable to be able to grab the cube with the simulated articulated hand
+            testObject.AddComponent<NearInteractionGrabbable>();
+
+            yield return new WaitForFixedUpdate();
+            yield return null;
+
+            TestHand hand = new TestHand(Handedness.Right);
+
+            yield return hand.Show(initialObjectPosition);
+            yield return null;
+
+            // grab the object
+            yield return hand.SetGesture(ArticulatedHandPose.GestureId.Pinch);
+            yield return new WaitForFixedUpdate();
+            yield return null;
+
+            float testRotation = 45;
+            const int numRotSteps = 30;
```

Comment:
Might want to do less to get tests to run faster. As Lukas suggested perhaps we should have  a default "num steps" amount that's like, 10 or less.

Reply:
I have a PR open for unifying those numSteps across tests, but doing that breaks a lot of them right now. So i would postpone that until after GA.
#5358

Reply:
will tune it down a bit for now - final steps will be decided by lukas clean up PR :)